service/chain/disk.go

```diff
@@ -107,21 +107,21 @@ func (d *Disk) Transactions(height uint64) ([]flow.Transaction, error) {
 		return nil, dps.ErrFinished
 	}
 
-	var transactions []flow.Transaction
+	var transactions []*flow.TransactionBody
```

Comment:
why not `make([]*flow.TransactionBody, 0, len(tt))`?

Reply:
I didn't really change that code, though your point is valid.